### DISCUSSION TITLE
HASS Add On won't install

### DISCUSSION THREAD

**Discussion by mhendu** [fuatakgun/eufy_security#189] [Q&A] [answered]
I'm running Hassio on Ubuntu. When I go to the Add On Store I'm able to add the repository, but when I click Install on the add-on it gives me a spinning circle for a second or so then stops, and the add-on is not installed. How do I proceed from here? Thanks in advance!

**Comment by fuatakgun** [comment]
I do not think it is an add on issue, please check supervisor logs to see more details. If there is an issue with add-on, it should start while starting not installing. Installing add-ons are part of docker process and managed by supervisor

**Comment by mhendu** [comment]
This is the error I receive:

21-12-28 13:31:12 ERROR (SyncWorker_2) [supervisor.docker.interface]
Can't install bropat/eufy-security-ws:0.6.0: 404 Client Error for
http+docker://localhost/v1.40/images/bropat/eufy-security-ws:0.6.0/json:
Not Found ("no such image: bropat/eufy-security-ws:0.6.0: No such
image: bropat/eufy-security-ws:0.6.0")

**Reply by fuatakgun** [reply]
hmm, i am not sure if i can help here any, I see that image is there (https://hub.docker.com/layers/bropat/eufy-security-ws/0.6.0/images/sha256-d24fed952e96effe1f0b2fa92063d93517b6b9a413dc9bbfc30d449adbbf2a4b?context=explore) and i was able to uninstall and install it back locally. Can you give it a try with a local docker instance rather than home assistant?

**Comment by mhendu** [comment]
My Linux knowledge is a bit limited - do you mean run this command from the
setup instructions?

docker run -it -e ***@***.*** -e PASSWORD=password_goes_here
-p 3000:3000 bropat/eufy-security-ws:latest

**Reply by fuatakgun** [reply]
Yes run it from command line but replace `latest` with `0.6.0`. are you using home assistant operating system or linux with supervised?

**Comment by mhendu** [comment]
OK will try that. I'm running Hassio on Ubuntu 18.04, 32-bit.

**Reply by fuatakgun** [reply]
Hmm i don't think you can run this command in hassio. Can you try to install second add on (rtsp server) to see if this is specific to first (Eufy) addon or general problem?

**Comment by mhendu** [comment]
I got a permission denied error so I had to run with sudo, then I get an
error 'no matching manifest for linux/386 in the manifest list entries.'
Does this mean this only works on x64 systems?

**Comment by mhendu** [comment]
I can add the repository for the RTSP server but get this error, which
prevents me from installing it:

This add-on is not compatible with the processor of your device or the
operating system you have installed on your device.

**Reply by fuatakgun** [reply]
Got it, apparently you are right, it is about OS related limitation. I am on armv7l.

**Comment by mhendu** [comment, marked as answer]
Maybe I should open a separate thread for this, but still trying to get installed. I set up Home Assistant Supervised via a Debian virtual machine on my Windows 10 (64-bit) desktop and copied over settings from my 32-bit computer. Runs much faster, still trying to get all devices ported over. I was able to get the add-on installed and the HACS integration set up, but when I go to Configuration / Integrations and try to install EufySecurity I get an error:

Config flow could not be loaded

Any idea how to fix this issue?

**Reply by mhendu** [reply]
The logs show this error associated with the above:

Error occurred loading configuration flow for integration eufy_security: cannot import name 'ConfigEntryAuthFailed' from 'homeassistant.exceptions' (/usr/src/homeassistant/homeassistant/exceptions.py)

**Reply by mhendu** [reply]
OK, after a bit of research looks like for whatever reason although this is a fresh install of Home Assistant it downloaded a version from April. I upgraded to the most recent version and was able to install. Success!
